test/example_data.dart

```diff
@@ -18,10 +19,52 @@ void _checkPositive(int? value, String description) {
   assert(value == null || value > 0, '$description should be positive');
 }
 
+////////////////////////////////////////////////////////////////
+// Error objects.
+//
+
 Object nullCheckError() {
   try { null!; } catch (e) { return e; } // ignore: null_check_always_fails
 }
 
+/// A Zulip API error with the generic "BAD_REQUEST" error code.
+///
+/// The server returns this error code for a wide range of error conditions;
+/// it's the default within the server code when no more-specific code is chosen.
+ZulipApiException apiBadRequest({
+    String routeName = 'someRoute', String message = 'Something failed'}) {
+  return ZulipApiException(
+    routeName: routeName,
+    httpStatus: 400, code: 'BAD_REQUEST',
+    data: {}, message: message);
+}
+
+/// The error for the "events" route when the target event queue has been
+/// garbage collected.
+///
+/// https://zulip.com/api/get-events#bad_event_queue_id-errors
+ZulipApiException apiExceptionBadEventQueueId({
+  String queueId = 'fb67bf8a-c031-47cc-84cf-ed80accacda8',
```

Comment:
Is this what a queue ID looks like? It differs from what we have in `eg.initialSnapshot`:
```dart
    queueId: queueId ?? '1:2345',
```

Reply:
Yeah. This comes from [/api/get-events](https://zulip.com/api/get-events#bad_event_queue_id-errors). We also use "abc:123" in another test. It might be good to have such a realistic value in test/example_data.dart to share.

Reply:
I see, and in an actual /api/register-queue response I see a similar value.

I guess it must have changed at some point. Pretty sure in the past (years ago) it had the other form.